src/test/java/snippets/SnippetsTests.java

```diff
@@ -322,13 +324,18 @@ public void nthElement_return_nth_element_from_end_when_n_is_less_than_0() throw
 
     @Test
     public void pick_should_pick_key_pairs_corresponding_to_keys() throws Exception {
-        Map<String, Integer> obj = new HashMap<>();
-        obj.put("a", 1);
-        obj.put("b", 2);
-        obj.put("c", 3);
+        
+        Map<String, Integer> obj = new LinkedHashMap<>();    
+        obj.put("a", 1);    
+        obj.put("b", 2);    
+        obj.put("c", 3);    
+      
+        Map<String, Integer> picked = Snippets.pick(obj, new String[]{"a", "c"});    
+        Map<String, Integer> expected = new LinkedHashMap<>(); 
+        expected.put("a", 1); 
+        expected.put("c", 3); 
+        Assertions.assertThat(picked).isEqualTo(expected); 
 
-        Map<String, Integer> picked = Snippets.pick(obj, new String[]{"a", "c"});
-        assertThat(picked).containsExactly(new SimpleEntry<>("a", 1), new SimpleEntry<>("c", 3));
```

Comment:
Why does this fail in the first place? Does it ever missed `a` or `c`, or does it select `b`? Or is the case that just the order differs? Is there `containsExactlyInAnyOrder`? That may be a much simpler fix.